### PR TITLE
Fixing Travis again

### PR DESCRIPTION
**Pull request by gabro**
Fingers crossed

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ jdk:
 stages:
   - name: test
   - name: release
-    if: (branch = master) AND (type = push OR tag IS present)
+    if: (branch = master) AND (type = push) OR (tag IS present)
```
Comment (olafurpg) [inline .travis.yml]: What's the precedence here?

```
((branch = master) AND (type = push)) OR (tag IS present)
(branch = master) AND ((type = push) OR (tag IS present))
```

Comment (laughedelic) [inline .travis.yml]: Probably unnecessary, but I would group master and push conditions

Comment (gabro) [reply .travis.yml]: years of computer science tell me `AND` associates stronger than `OR`, but I won't take my chances and I'll add the parens :D

Comment (gabro) [reply .travis.yml]: https://github.com/scalameta/metals/pull/245#discussion_r179508207

Comment (olafurpg) [reply .travis.yml]: I've written two Scala code formatters handling infix operator precedence and I can never remember how it goes 😅

Comment (gabro) [reply .travis.yml]: It's super easy in Scala: you just need to remember a fixed list of 500 symbols and their precedences 😝

**Pull request merged**